air-quality-backend/system_tests/api_suite/forecast_api_seeded_data_tests.py

```diff
@@ -0,0 +1,387 @@
+import datetime
+import pprint
+import pytest
+from air_quality.database.forecasts import Forecast
+from system_tests.utils.api_utilities import (
+    format_datetime_as_string,
+    get_list_of_keys,
+    get_forecast,
+    setup_purge_database_and_seed_with_test_data,
+)
+
+# Test Data
+test_city_1_input_data: Forecast = {
+    "forecast_valid_time": datetime.datetime(2024, 6, 10, 3, 0, 0),
```

Comment:
Do we need to worry about timezones here

Reply:
In the test data and the parameters with dates I have used tzinfo=datetime.timezone.utc, however have had to format the strings to "%Y-%m-%dT%H:%M:%S+00:00" as the response timezone is formatted to have +00:00, whereas the tzinfo=datetime.timezone.utc has 0000. Would this be ok?